libs/platform/src/lib/components/form/multi-input/multi-input.component.ts

```diff
@@ -0,0 +1,137 @@
+import { EventEmitter, forwardRef, Input, Output, TemplateRef } from '@angular/core';
+import { Host, Optional, Self, SkipSelf } from '@angular/core';
+import { ViewContainerRef } from '@angular/core';
+import { ViewChild } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ViewEncapsulation } from '@angular/core';
+import { NgControl, NgForm } from '@angular/forms';
+import { TokenizerComponent } from '@fundamental-ngx/core';
+import { KeyUtil } from '@fundamental-ngx/core';
+import { BaseInput } from '../base.input';
+import { FormFieldControl } from '../form-control';
+import { FormField } from '../form-field';
+import { InputGroupComponent } from '../input-group/input-group.component';
+import { InputType } from '../input/input.component';
+
+@Component({
+    selector: 'fdp-multi-input',
+    templateUrl: './multi-input.component.html',
+    styleUrls: ['./multi-input.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: FormFieldControl,
+            useExisting: forwardRef(() => PlatformMultiInputComponent),
+            multi: true
+        }
+    ]
+})
+export class PlatformMultiInputComponent extends BaseInput {
+    /** Whether the combobox is opened. */
+    isOpen = false;
+
+    oldPlaceHolder: string;
+    tokenLength = 0;
+
+    @Input()
+    type: InputType;
+
+    /** Selected dropdown items. */
+    @Input()
+    selected: any[] = [];
+
+    @Input()
+    listTemplate: TemplateRef<any>;
+
+    @ViewChild('listTemplate', { read: ViewContainerRef }) outletRef: ViewContainerRef;
+
+    /** @hidden */
+    @ViewChild(TokenizerComponent)
+    tokenizer;
+
+    /** @hidden */
+    @ViewChild(InputGroupComponent)
+    inputComponent;
+
+    /** @hidden Emits event when the menu is opened/closed */
+    @Output()
+    isOpenChange: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+    public updateSelectedListVariables = [];
+
+    private _listVariable: any;
```

Comment:
if possible, we need to avoid usage of any.

Reply:
these values are passed from the list to the multiInput component, we can't make it specific data type